clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -343,3 +345,17 @@ export const CustomCreatableMultiSelect = ({
     </FormControl>
   );
 };
+
+export const CustomTextArea = ({
+  textAreaProps,
+  ...props
+}: { textAreaProps?: TextareaProps } & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
+  return (
+    <FormControl isInvalid={isInvalid}>
+      <Textarea {...field} size="sm" mb={2} {...textAreaProps} />
+      {isInvalid ? <FormErrorMessage>{meta.error}</FormErrorMessage> : null}
```

Comment:
The error message is pretty 😜 when the input is parseable yaml but not a valid dataset:
![Screen Shot 2022-06-30 at 14 04 55](https://user-images.githubusercontent.com/2236777/176780320-aa6367f6-88a6-4df6-ac15-750d99fafe43.png)
I guess that's coming from the backend.